README.md

```diff
@@ -58,11 +58,12 @@ $ minikube ssh -- test -e /dev/kvm \
 Now you are finally ready to deploy KubeVirt using our operator (comparable to an installer):
 
 ```bash
-$ kubectl apply -f https://github.com/kubevirt/kubevirt/releases/download/v0.24.0/kubevirt-operator.yaml
+$ export KUBEVIRT_LAST_VERSION=$(curl -s https://api.github.com/repos/kubevirt/kubevirt/releases/latest | jq -r .tag_name)
```

Comment:
This logic sadly does not work, as GH does not do semver based last, but only time based last

Reply:
`export KUBEVIRT_VERSION=$(curl -s https://api.github.com/repos/kubevirt/kubevirt/releases|grep tag_name|sort -V | tail -1 | awk -F':' '{print $2}' | sed 's/,//' | xargs | cut -d'-' -f1)`

That's how we do it at Quickstart, sort -V takes care of semver sorting

Reply:
Yes. I had that in the past as well, but went back to static version numbres to make copy and pasting easy and less error prone

Reply:
The static version is a pain as somebody has to update the README  every time there is a release, is that what you prefer to have?

Reply:
@fabiand how would you like to have it? static version or automatic version?